python/ray/experimental/slib/examples/wordcount.py

```diff
@@ -0,0 +1,44 @@
+import time
```

Comment:
Many of the same comments from the previous example apply to this file as well.

- alphabetical order for imports
- don't do `import *`
- `'` -> `"`
- avoid abbreviations for variable names
- don't use `\` 
- use logger instead of print 
- `from __future__ ...` imports